src/appleseedmaya/typeids.h

```diff
@@ -110,7 +110,10 @@ enum AppleseedMayaTypeIds
     AsManifold2DNodeTypeId          = 0x00127a05,   // 1210885
     AsManifold3DNodeTypeId          = 0x00127a06,   // 1210886
     AsAscCdlNodeTypeId              = 0x00127a07,   // 1210887
-    AsInvColorNodeTypeId            = 0x00127a08,   // 1210888
+    AsInvertColorNodeTypeId         = 0x00127a08,   // 1210888
+    AsBlendNormalsNodeTypeId        = 0x00127a09,   // 1210889
```

Comment:
Changing the value of `AsBlendNormalsNodeTypeId` here; is this intentional, and will this preserve backward compatibility?

Reply:
It's a new node so it poses no problem, but i usually reserve IDs in advance as a reminder and general roadmap as well.